Assets/Art/Textures/VFX/Shared/Flames/Flame_Welding.mat

```diff
@@ -0,0 +1,98 @@
+%YAML 1.1
```

Comment:
.mats go under Assets>Content

To put it simply:
- assets made in Unity go under Content
- assets made in outside of Unity go in Art

Reply:
AH - yes sorry
@joaoburatto ill revert that change, my bad! that was me being dumb
@cosmiccoincidence i will fix that :)

ill make these changes tomorrow because im going to bed now

Reply:
Resolved in latest commits